From a2d1fa48c6d9b29f3084d063f4a99a8af2d82aef Mon Sep 17 00:00:00 2001
From: oliverwen <oliverwen@google.com>
Date: Tue, 28 Jun 2016 17:05:07 -0700
Subject: [PATCH] CHROMIUM: ath10k: Fix smart antenna compilation issues

With 4.2 wireless stack, smart antenna code doesn't compile cleanly.
So fix it

TEST=building the 4.2 wireless stack should pass

BUG=chrome-os-partner:51964
CQ-DEPEND=CL:341526

Change-Id: I3ab4ef2ed43c7a586a891897ab0e9d31bde3aa15
Signed-off-by: oliverwen <oliverwen@google.com>
Reviewed-on: https://chromium-review.googlesource.com/341565
Commit-Ready: Yixiang Li <yixiang@google.com>
Tested-by: Oliver Wen <oliverwen@chromium.org>
Reviewed-by: Yixiang Li <yixiang@google.com>
---
 drivers/net/wireless/ath/ath10k/smart_ant.c | 2 +-
 1 files changed, 1 insertions(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath10k/smart_ant.c b/drivers/net/wireless/ath/ath10k/smart_ant.c
index 9fccf07b960..1fe4c6bc389 100644
--- a/drivers/net/wireless/ath/ath10k/smart_ant.c
+++ b/drivers/net/wireless/ath/ath10k/smart_ant.c
@@ -1545,7 +1545,7 @@ ath10k_smart_ant_proc_rx_feedback(struct ath10k *ar,
 		RX_ATTENTION_FLAGS_LAST_MPDU))
 		return;
 
-	sa_info->rx_antenna = __le32_to_cpu(rx_desc->ppdu_end.common.info0) &
+	sa_info->rx_antenna = __le32_to_cpu(rx_desc->ppdu_end.qca988x.info0) &
 				ATH10K_RX_ANT_MASK;
 }
 
-- 
2.43.0

